sn_transfers/src/wallet/local_store.rs

```diff
@@ -144,11 +151,23 @@ impl LocalWallet {
 
     /// To remove a specific spend from the requests, if eg, we see one spend is _bad_
     pub fn clear_specific_spend_request(&mut self, unique_pub_key: UniquePubkey) {
+        if let Err(error) = self.remove_cash_notes_from_disk(vec![&unique_pub_key]) {
+            warn!("Could not clean spend {unique_pub_key:?} due to {error:?}");
+        }
+
         self.unconfirmed_spend_requests
             .retain(|signed_spend| signed_spend.spend.unique_pubkey.ne(&unique_pub_key))
     }
 
-    pub fn clear_unconfirmed_spend_requests(&mut self) {
+    /// Once spends are verified we can clear them and clean up
+    pub fn clear_confirmed_unconfirmed_spend_requests(&mut self) {
```

Comment:
the function only removes `unconfirmed_spend_requests` ?

Reply:
im not sure what the question is here? It's those that should be cleaned up, or?

Reply:
I mean the function name (and the comment) doesn't matches the internall calls.

Reply:
Ah, just notice the code has been updated, will mark this as resolved.